china/2020_EKS_Launch_Workshop/步骤1-准备实验环境.md

```diff
@@ -68,9 +68,9 @@ sudo mv ./eksctl /usr/local/bin
 #kubectl
 https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html
 #macOS
-curl -o kubectl https://amazon-eks.s3-us-west-2.amazonaws.com/1.15.10/2020-02-22/bin/darwin/amd64/kubectl
```

Comment:
this is fix md format issue ?  Im not get your poing   :<

Reply:
It is tool automatically fix issue.  I manually fix it.